03 Writing Algorithms/03 Securities/99 Asset Classes/07 Futures/04 Market Hours/16 INDIA/01 Introduction.html

```diff
@@ -1 +0,0 @@
-<p>This page shows the trading hours, holidays, and time zone of the India Future market.</p>
```

Comment:
The repo [currently has](https://github.com/QuantConnect/Documentation/tree/master/03%20Writing%20Algorithms/03%20Securities/99%20Asset%20Classes/07%20Futures/04%20Market%20Hours) two directories for INDIA data, `16 INDIA` and `17 INDIA`. Can this PR fix this so that there is only 1 directory for India data?